### PR TITLE
refactor service to use postgres (from mysql)

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -3,17 +3,17 @@ APP_COMPONENT=api
...
+READ_DB_USER=postgres
 READ_DB_PASS=lol123
-READ_DB_HOST=mysql
-READ_DB_PORT=3306
+READ_DB_HOST=postgres
+READ_DB_PORT=5432
 READ_DB_NAME=beatmap_ids
-WRITE_DB_DRIVER=mysql
-WRITE_DB_USER=root
+WRITE_DB_DRIVER=postgres
```
Comment (cmyui) [inline .env.example:12]:
```suggestion
WRITE_DB_DRIVER=postgresql
```

File: .env.example [hunk 2]
```diff
@@ -3,17 +3,17 @@ APP_COMPONENT=api
 APP_HOST=0.0.0.0
 APP_PORT=80
 LOG_LEVEL=10
-READ_DB_DRIVER=mysql
-READ_DB_USER=root
+READ_DB_DRIVER=postgres
```
Comment (cmyui) [inline .env.example:6]:
```suggestion
READ_DB_DRIVER=postgresql
```

Comment (cmyui) [reply .env.example:6]: thanks for this one @retrocraft lol

Comment (RetroCraft) [reply .env.example]: LMAO i can't believe nobody noticed until they made a breaking change over a decade later

**Pull request merged**